src/components/Actions.tsx

```diff
@@ -92,6 +92,9 @@ export const SelectedShapeActions = ({
       {showChangeBackgroundIcons && (
         <div>{renderAction("changeBackgroundColor")}</div>
       )}
+      {useExcalidrawActionManager()
+        .getCustomActions({ elements: targetElements })
+        .map((action) => renderAction(action.name))}
```

Comment:
this is hardcoding all custom actions to be rendered in the properties sidebar which is arbitrary. And we will be removing component rendering from actions anyway: https://github.com/excalidraw/excalidraw/issues/6075

As such, rendering will need to be handled by host once we add support for rendering stuff to all manner of components such as the properties sidebar.

Reply:
`ActionManager.getCustomActions` filters out custom actions, including by predicates and action guards in `ActionManager.isActionEnabled`.  I suppose the caller of `getCustomActions` should have the discretion to call `isActionEnabled`.

Reply:
I don't mean `getCustomActions` here, but the rendering into the sidebar.

Reply:
I have removed this.